### ISSUE TITLE
Only scraping main page

### ISSUE THREAD

**Issue by dueni0** [openzim/mwoffliner#425]
With the command:

mwoffliner --verbose --mwUrl=http://myhostname/ --adminEmail=user@myhostname --mwWikiPath=index.php --mwApiPath=api.php/

I get the error:
```
Dumping stylesheets...
Cache hit for http://myhostname/index.php/ (/root/cac/mywikiname_de/251af7f3cae47392c9cf)
Downloading CSS from http://myhostname/load.php?debug=false&lang=de&modules=ext.smw.style|ext.smw.tooltip.styles&only=styles&skin=vector
Downloading http://myhostname/load.php?debug=false&lang=de&modules=ext.smw.style|ext.smw.tooltip.styles&only=styles&skin=vector...
Downloading CSS from http://myhostname/load.php?debug=false&lang=de&modules=ext.visualEditor.desktopArticleTarget.noscript|mediawiki.legacy.commonPrint%2Cshared|mediawiki.sectionAnchor|mediawiki.skinning.interface|skins.vector.styles&only=styles&skin=vector
Downloading http://myhostname/load.php?debug=false&lang=de&modules=ext.visualEditor.desktopArticleTarget.noscript|mediawiki.legacy.commonPrint%2Cshared|mediawiki.sectionAnchor|mediawiki.skinning.interface|skins.vector.styles&only=styles&skin=vector...
TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received type object
    at assertPath (path.js:39:11)
    at Object.basename (path.js:1297:5)
    at /usr/local/lib/node_modules/mwoffliner/lib/mwoffliner.lib.js:1608:41
    at async.retry (/usr/local/lib/node_modules/mwoffliner/lib/Downloader.js:157:7)
    at /usr/local/lib/node_modules/mwoffliner/node_modules/async/lib/async.js:676:51
    at /usr/local/lib/node_modules/mwoffliner/node_modules/async/lib/async.js:726:13
    at /usr/local/lib/node_modules/mwoffliner/node_modules/async/lib/async.js:52:16
    at /usr/local/lib/node_modules/mwoffliner/node_modules/async/lib/async.js:264:21
    at /usr/local/lib/node_modules/mwoffliner/node_modules/async/lib/async.js:44:16
    at /usr/local/lib/node_modules/mwoffliner/node_modules/async/lib/async.js:723:17
```

issue reopened from #335

**Comment by ISNIT0** [comment]
@dueni0 Any chance we can see the actual host/wiki? It's quite helpful for reproducing and debugging.

**Comment by dueni0** [comment]
the wiki is internal only

**Comment by ISNIT0** [comment]
This should do it:
```
git clone https://github.com/openzim/mwoffliner.git
cd mwoffliner
npm i
./bin/mwoffliner.script.js --verbose --mwUrl=http://myhostname/ --adminEmail=user@myhostname --mwWikiPath=index.php --mwApiPath=api.php/
```

**Comment by dueni0** [comment]
it works (after installing the librsvg2-bin package), but only the wiki mainpage gets dumped into the .zim file

**Comment by dueni0** [comment]
there is another error in the debug output:

` pngquant: unrecognized option '--strip'`

it seems that the correct parameter would be '-strip'

**Comment by ISNIT0** [comment]
Could you send the following information please?
- Operating System
- `node --version`
- `pngquant -v`

**Comment by dueni0** [comment]
os: Ubuntu 16.04.5 LTS 
node: v10.12.0
pngquant: 2.5.0 (June 2015)

**Comment by ISNIT0** [comment]
Could you try updating pngquant please?

**Comment by dueni0** [comment]
pngquant is 2.12.2 (November 2018) now, the --strip error is gone but the generated zim file still contains the main page only

**Comment by ISNIT0** [comment]
Thanks @dueni0, I'll take a look when I can :)

**Comment by kelson42** [comment]
@dueni0 is now impacted by a major regression in git master version. I'm impacted too. I put the ticket in 1.6 as the last problem need to be fixed with hight priority without releasing 1.6 anyway.
